ui/src/domain/identity.ts

```diff
@@ -0,0 +1,38 @@
+export type Identifier = string;
```

Comment:
sorry, but I can't understand what the problem is

Reply:
We don't need a type `Identifier` that equals `string`.

This adds unnecessary cognitive load since I need to go to the type `Identifier` to realize it is a `string`.

Wherever we use `Identifier` we should use `string` instead.